CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,111 @@
+# Contributor Ladder Template
```

Comment:
I think we should remove the wording that this is a template and adjust it to the needs of the project.